.travis.yml

```diff
@@ -8,29 +8,14 @@ go:
 before_install:
   - go get github.com/mattn/goveralls
   - go get golang.org/x/tools/cmd/cover
-  - go get github.com/modocache/gover
 
 env:
   global:
     secure: IgtNHjaBYoiMXsLjyc/ZsciSTb027D+24L9iHEWXnH+L82tEZUhYUDI1hZjYtu1WXtfjb9taIXE/wTGR6ET5nDaoF1mmNXUFSzFDciIsLw7HDANcAGiUwY6D4Bu9sbVKnEskWc2v1dlEAiIVyGGbu2opV1eq2DcijidJ4bXdHotDuOTRiTZwgsE1Nou59zg+mpw3XliGmM5IRmZ60bKSw33aRE3D8H/pv98Eorfnq6UNHOBVA1SLTfbQoX0rGTLccDKqBEFUlXLURYiJVGguQ331lo4c61gOMHaF0GHaarYEbR6uRPFx7oKidGFVOm8l8fqq84Bimo9JIljAcVR+2YRln++3yw8VYCoMSL5+9O15LVBX98EFqKernMthHV0OqXu3wUVMcuA4/RCbBNQ8pCnLHQm/siowhNps0DDjdZjaWNB6dHRBP+4A7Cm4AEhveWxLUStsVLCQueJKxmEqYIKRUtXE6otz2yYHbgLQoPFMAOomjov9enMtJ66vgHwQ50Dwx6adjN29k3NxPQCBlR/iLhFRm247/C8vFCbnzMa/9NwJa0wNs7/++qBDeAAngp7kUc3mDK6lvgmio3SDHI5L/ic8TRFYZX/eNRo7txax1vN8Smqmy1Tl2rWSt8XJmop556grWmZ1DcY1Md8mo6I8BrnJfljfjuoQcI0TmvM=
 
 script:
   - test -z "$(go fmt ./...)"
-  - go test -coverprofile=cgo.coverprofile ./client/cgo
-  - go test -coverprofile=crypto.coverprofile ./crypto
-  - go test -coverprofile=ed25519.coverprofile ./crypto/ed25519/edwards25519
-  - go test -coverprofile=extra25519.coverprofile ./crypto/ed25519/extra25519
-  - go test -coverprofile=sign.coverprofile ./crypto/sign
-  - go test -coverprofile=vrf.coverprofile ./crypto/vrf
-  - go test -coverprofile=merkletree.coverprofile ./merkletree
-  - go test -coverprofile=utils.coverprofile ./utils
-  - go test -coverprofile=protocol.coverprofile ./protocol
-  - go test -coverprofile=keyserver.coverprofile ./keyserver
-  - go test -coverprofile=bots.coverprofile ./bots
-
-after_success:
```

Comment:
Done in https://github.com/coniks-sys/coniks-go/pull/85/commits/1ca5702c0e21611939af6cb1f0c400e149e318a4